src/pages/index.html

```diff
@@ -1,29 +1,332 @@
+        >
+          {{headerDataEn.btn.name}}
+        </a>
+      </div>
+      <div class="row justify-content-center animated delay fadeIn">
+        <div class="col-sm-10 col-12">
+          <div class="header-container--dashboard-box">
+            <img src="../../assets/images/main/dashboard.png" alt="dashboard" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
 
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-3">
-                {% include 'component/card.html' %}
-                {{ cards.mainCard({type: 'text', name: 'name'}) }}
-                {{ cards.mainCard({type: 'number', name: 'phone'}) }}
+<section>
+  <div class="container">
+    <div class="users-container d-flex flex-column">
+      <h1 class="users-container--title">
+        {{ homeDataEn.users.title.main }}
+        <span class="users-container--title--marker animated delay fadeIn">
+          {{ homeDataEn.users.title.marker }} </span
+        >.
+      </h1>
+      <div
+        class="users-container--list d-flex justify-content-center align-items-center flex-wrap"
+      >
+        {% for i in range(1, 6) %}
+        <img src="../assets/images/users/user-{{ i }}.png" alt="user-logo" />
+        {% endfor %}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="container">
+    <div class="services-container d-flex flex-column align-items-center">
+      <div class="d-flex flex-column align-items-center">
+        <h1
+          class="services-container--title d-flex flex-column align-items-center text-center"
+        >
+          <span> {{ homeDataEn.services.title.main }} </span>
+          <span class="services-container--title--marker animated delay fadeIn">
+            {{ homeDataEn.services.title.marker }}
+          </span>
+        </h1>
+        <p class="services-container--info text-center">
+          {{ homeDataEn.services.info }}
+        </p>
+      </div>
+      <div class="services-container--list">
+        <div class="container">
+          <div
+            class="row justify-content-center align-items-stretch gap-x-32 gap-y-40"
+          >
+            {% for item in homeDataEn.services.list %}
+            <div class="col-lg-3 col-sm-6 col-12">
+              <div
+                class="services-container--list--item-box d-flex flex-column align-items-start h-100 animated rubberBand"
+              >
+                <span class="services-container--list--item-box--icon">
+                  {{ item.icon | safe }}
+                </span>
+                <h2 class="services-container--list--item-box--title">
+                  {{ item.title }}
+                </h2>
+                <p class="services-container--list--item-box--info">
+                  Security testing is a type of software testing that is focused
+                  determining if an information.
+                </p>
+              </div>
             </div>
```

Comment:
You should've added transition effect to make the hover effect smoother

Reply:
done.